templates/parachain/pallets/template/src/lib.rs

```diff
@@ -38,19 +86,30 @@ pub mod pallet {
 	#[pallet::pallet]
 	pub struct Pallet<T>(_);
 
+	/// A struct to store a single block-number. Has all the right derives to store it in storage.
+	/// <https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/reference_docs/frame_storage_derives/index.html>
```

Comment:
I guess this link will start working once we publish it?

Reply:
Exactly, it will work after merging this PR. I might have to remote it to make the CI pass though, as the link checker is mandatory.

Reply:
The link checker is optional since it is not marked as `Required` 😉

<img width="890" alt="Screenshot 2024-07-08 at 14 26 20" src="https://github.com/paritytech/polkadot-sdk/assets/10380170/5f964648-c6d3-485f-b9c2-d798d8cdd789">